api/views/location.py

```diff
@@ -10,3 +12,29 @@ class LocationList(generics.ListCreateAPIView):
 class LocationDetail(generics.RetrieveUpdateDestroyAPIView):
     queryset = Location.objects.all()
     serializer_class = LocationSerializer
+
+class NearbyLocations(generics.ListAPIView):
+    serializer_class = LocationSerializer
+    
+    def get_queryset(self):
+        params = self.request.query_params
+        
+        try:
+            latitude = float(params.get('coordinates').split(',')[0])
```

Comment:
instead of splitting coordinates would it be easier to do ?latitude=123&longitude=456 ?
also i think there should be a count param to limit the number of results returned at once

Reply:
I think it makes more sense to have both in a single param cuz one should only appear with the other anyway, also google's nearbysearch api uses the same tuple format for coordinates

True I'll add the count param